app/assets/javascripts/discourse/app/lib/plugin-connectors.js

```diff
@@ -71,21 +81,77 @@ export function expireConnectorCache() {
   _connectorCache = null;
 }
 
+class ConnectorInfo {
+  #_componentClass;
+  #_templateOnly;
```

Comment:
`#` **and** `_`? 😃 I have a suspicion those fields are private 🤪 (I think we can drop _ when using priv fields?)